.eslintrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  // See https://github.com/torchbox/eslint-config-torchbox for rules.
+  extends: 'torchbox',
```

Comment:
As per stylelint comment. Not sure if it's worth using Wagtail's config instead. 

https://github.com/wagtail/eslint-config-wagtail